rest/src/main/java/com/netflix/conductor/rest/rbac/RbacHttpFilter.java

```diff
@@ -73,9 +73,11 @@ public void doFilter(
 
         HttpServletRequest request = (HttpServletRequest) servletRequest;
         HttpServletResponse response = (HttpServletResponse) servletResponse;
+        List<String> noRbacValues = List.of("/health", "/v3/api-docs");
 
-        final String healthCheck = "health";
-        if (request.getRequestURI().contains(healthCheck)) {
+        boolean isRbacless =
```

Comment:
rename variable to ignoreRbac or something similar.

Reply:
Renamed.